site/zenodo_rdm/migrator/__init__.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+from stream import RecordStream
+
+
+if __name__ == "__main__":
+
+    filename = sys.argv[1]
+    cleanup = len(sys.argv) > 2  # if there is something we assume is True/--cleanup
+    stream = RecordStream(
+        filename=filename,
+        db_uri="postgresql://zenodo:zenodo@localhost:5432/zenodo",
```

Comment:
major: both are hardcoded to my current setup. How do we want to run this? click? or for now I can add `argv[2]` and `argv[3]`?

Reply:
Discussed IRL:

- we'll keep the `__main__` + `argv` usage in the instances for now
- we configure the classes for each stream (users, communities, records) in Python in the instance
- we introduce a YAML config to pass specific parameters to the ETL classes (e.g. `db_uri`, `filename` for the JSONL extract, `validate=True`, `cleanup=True`, etc.)